pkg/cmd/roachtest/cluster.go

```diff
@@ -2196,7 +2197,9 @@ func (c *cluster) RunE(ctx context.Context, node nodeListOption, args ...string)
 	physicalFileName := l.file.Name()
 	l.close()
 	if err != nil {
-		_ = os.Rename(physicalFileName, strings.TrimSuffix(physicalFileName, ".log")+".failed.log")
+		failedPhysicalFileName := strings.TrimSuffix(physicalFileName, ".log") + ".failed"
```

Comment:
this turns `foo.log` into `foo.failed`, so the comment no this method is inaccurate. It should be

> // have an additional marker file with a `.failed` extension instead of `.log`.

Reply:
Drat. I missed updating the comment after changing the extension to just `.failed`. It's updated now.